STYLE_GUIDE.md

```diff
@@ -0,0 +1,88 @@
+# Style Guide
+
+This document outlines the style guide for the `epinowcast` package. This guide is a work in progress and will be updated as the package evolves. We welcome contributions to this guide and encourage you to raise issues or submit PRs if you have any suggestions.
+
+In addition to this guide we also follow the [tidyverse style guide](https://style.tidyverse.org/). This guide is a subset of the `tidyverse` style guide and outlines the additional style requirements for the `epinowcast` package.
+
+## Naming conventions
+
+- We use a `enw_` prefix to delineate functions that are exported by the `epinowcast` package. This is to avoid conflicts with other packages and to make it clear to users which functions are part of the package.
```

Comment:
This should maybe be added to CI checks. The simplest option is likely to check the `NAMESPACE`

Reply:
So that is a good idea but we haven't been quite consistent (mostly recently) so there are a few functions that don't do this (I guess we could right exceptions). There is an open issue to address the prefix more generally as when it was discussed at previous meetings the universal feeling was one of dislike.

Reply:
turn it into an issue then (adding the CI checks + renaming to get in compliance with them)?

Reply:
I don't think we want to. I think we want to discuss the prefix more generally first (which is an open issue).

Reply:
So I am just going to resolve this but happy to have an issue on this (needs to link to the current prefix discussion I think)